include/pacmod3_dbc3_ros_api.h

```diff
@@ -35,6 +35,7 @@ class Dbc3Api : public DbcApi
 {
 public:
   Dbc3Api();
+  virtual ~Dbc3Api() = default;
 
   std::shared_ptr<void> ParseAccelAuxRpt(const cn_msgs::Frame& can_msg) override;
```

Comment:
Just return `void` is good enough. Any particular reason to return a shared ptr to void?

Reply:
The reason for this is because the type of message returned will be different depending on ROS1 or ROS2.

Reply:
Clarification: It's not because of ROS1/ROS2 differences, but it's to make the code easier on the driver side. It's not part of this PR, but there is a map of functions stored like this based on CAN ID:

```
std::map<uint32_t, std::function<std::shared_ptr<void>(const can_msgs::Frame&)>> parse_functions;
```

this is only possible with the same function signatures.